src/test/java/org/mule/extension/db/integration/select/SelectTestCase.java

```diff
@@ -87,6 +87,7 @@ public void maxRows() throws Exception {
 
   @Test
   public void emptyResult() throws Exception {
+    executeWithDataSources(dataSource -> testDatabase.truncateSpaceshipTable(dataSource.getConnection()));
```

Comment:
I deleted it in XML, so maybe that can be removed, if you do it here?

Reply:
Yes, I told Mariano that I hadn't seen the deletion in the flow. I'm removing that from the xml right now